pom.xml

```diff
@@ -151,24 +151,24 @@ THE SOFTWARE.
       <dependency>
         <groupId>junit</groupId>
         <artifactId>junit</artifactId>
-        <version>4.11</version>
+        <version>4.12</version>
```

Comment:
Yes, it's strange. I fill that some plugins that will switch on this core may get some issues. But then can override dependencies.